portfolio/src/main/webapp/css/main.css

```diff
@@ -274,9 +274,98 @@ html,h1,h2,h3,h4,h5,h6,a {
   background: none;
   border: none;
   outline: none;
-}
+} 
 
 /* maps style */
+.dessert-map {
+  margin: 4em 0;
+  padding: 1em;
+  position: relative;
+}
+
+.dessert-map h1 {
+  color: #5b5b57;
+  margin: 2em;
+}
+
+.dessert-map h3 {
+  color: #5b5b57;
+  justify-content: center;
+  margin: 2em;
+}
```

Comment:
Sometimes it's useful to combine selectors that have a lot of the same properties. Like this:

```
.dessert-map h1, .dessert-map h3 {
  color: #5b5b57;
  margin: 2em;
}

.dessert-map h3 {
  justify-content: center;
}
```

It's up to you if you think that makes more logical sense in this case.